client/src/components/pages/common/Login/index.js

```diff
@@ -1,5 +1,125 @@
 import React from 'react';
+import './index.css';
+import { ThemeProvider } from '@material-ui/core/styles';
+import axios from 'axios';
+import TextField from '@material-ui/core/TextField';
+import { Link } from 'react-router-dom';
+import Theme from './theme';
+import SimpleCard from '../../../common/card';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import loginValidate from '../../../../utils/application/LoginValidation';
 
-const Login = () => <h1>Login</h1>;
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: null,
+      password: null,
+      role: null,
+      message: '',
+    };
+  }
+
+  handleError = () => {
+    window.location.href = '/404';
+  };
+
+  throwMessage = (message) => {
+    this.setState({ message });
+  };
+
+  login = () => {
+    this.setState({ message: '' });
+    const { email, password } = this.state;
+    loginValidate({
+      email,
+      password,
+    }).then((result) =>
+      result
+        ? axios
+            .post('/api/v1/login', {
+              email,
+              password,
+            })
+            .then((response) => {
+              this.setState({ role: response.data.role }, () => {
+                this.redirect();
```

Comment:
instead of using redirect you can use `history`  for example : 
```suggestion
           const { history: { push }} = this.props;
            push('/');
```